src/components/Tooltip.vue

```diff
@@ -14,6 +14,10 @@ export default {
       default() {
         return {};
       }
+    },
+    icon: {
+      type: String,
+      default: 'fa fa-lg fa-info-circle'
```

Comment:
We don't need the `fa-lg` any more. And set this so that `<Tooltip>` only needs to be called with the icon name, e.g. `<Tooltip icon="fa-info-circle">`. The `<Tooltip>` component can take care of adding the `fa` class.

```suggestion
      default: 'fa fa-info-circle'
```

Reply:
Approved by mistake